cub/test/catch2_test_thread_sort.cu

```diff
@@ -84,55 +89,34 @@ void Test()
   {
     thrust::sequence(data_source.begin(), data_source.end());
     thrust::shuffle(data_source.begin(), data_source.end(), re);
```

Comment:
suggestion: We can use our input generate here, e.g., to generate and test two different inputs: `c2h::gen(CUB_SEED(2), data_source);`

I had initially thought we were relying on generating unique key values, but we aren't since `std::uint8_t` overflows after 256 `elements`. Hence, I think using `c2h::gen` should work just as well.

Reply:
Great suggestion, applied!